bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/admin/MaintenanceHandler.java

```diff
@@ -266,4 +273,16 @@ public Response affectedBy( @ApiParam( "The store key" ) @PathParam( "key" ) fin
         return response;
     }
 
+    @ApiOperation( "Batch delete files under the given package store (type/name) and paths." )
+    @ApiResponse( code=200, message = "Batch delete operation finished." )
+    @ApiImplicitParam( name = "body", paramType = "body",
+                    value = "JSON object, specifying storeKey and paths, the option trackingID is not supported in this API.",
+                    required = true, dataType = "org.commonjava.indy.model.core.DemoteRequest" )
+    @Path("/content/batch/demote")
+    @POST
+    public Response doDelete( final DemoteRequest request )
```

Comment:
It seems like we need to decide whether to call this batch delete or demote...there's a mixture

Personally, I'd vote for batch-delete, since promotion is a set of features that are fairly well contained now, and this will spread that notion across the codebase.

Reply:
+1 for batch/delete

Reply:
I will update that later. :) thanks all.